app/controllers/journeys_controller.rb

```diff
@@ -1,10 +1,14 @@
 class JourneysController < ApplicationController
+  include NotificationSubscriptionsHelper
+
   def show
     @journey = Journey.published.find_by!(slug: params[:id])
     @next_step = @journey.steps.order(:position).first
 
     load_newest_user_journey(current_user, @journey)
 
+    @journey.description = @journey.description.gsub(/(?<=<insert_form>).*?(?=<\/insert_form>)/) {|s| render_notification_subscription_component_to_string([s])} if @journey.description.include? '<insert_form>'
```

Comment:
Neni ten `if` zbytocny?

Reply:
Ja by som inac toto presunul do Modelu na nejaky novy field, aby si nemusel pametat to replacnut vsade.

Reply:
Helper je lepsie miesto, to by som nechal tak.

Reply:
Asi by som to mergol s https://github.com/slovensko-digital/navody.digital/pull/229 a implementoval s **nokogiri** cez `CustomComponentsHelper` v https://github.com/slovensko-digital/navody.digital/pull/229 , kedze uz zaciname mat podobnu funktionalitu pre viacere veci. Co myslite?

Reply:
@smith558 dobry napad, CustomComponentsHelper uz je mergenuty v mastri.

Reply:
Idem na to